server/routes/forms/get.ts

```diff
@@ -24,8 +27,6 @@ export type FormGetRequestHandler<
   {
     forms: Forms
     submittedForm: Forms[keyof Forms] | null
-  } & {
-    user: Express.User
```

Comment:
app-specific types that are expected to be available to most request handler can now go into "server/@types/index.d.ts"